docs/getting-started/migrate/index.md

```diff
@@ -51,7 +52,32 @@ if [[ -f /HOME_DIR/.shopify-cli/shopify.sh ]]; then source /HOME_DIR/.shopify-cl
 [ -f "/HOME_DIR/.shopify-app-cli/shopify.sh" ] && source "/HOME_DIR/.shopify-app-cli/shopify.sh"
 ```
 
-Deleting or commenting out the relevant line in your shell profile will remove `shopify` as a command. You may need to reload your shell.
+Deleting or commenting out the relevant line in your shell profile will remove `shopify` as a command.
+
+### 3. Reload your terminal
+
+For the changes above to take effect, exit your terminal, and start a new one.
+
+To check that the previous Shopify App CLI is removed:
+
+```console
+$ which shopify
+shopify not found
+$
+```
+
+> Note: if the `which shopify` command above returns with a pathname, try running:
+> 
+> ```console
+> $ shopify
+> ```
+>
+> If the response is:
+> ```console
+> shopify command is no longer bundled with shopify_api.
+> if you need these tools, install the shopify_cli gem
+> ```
+> then you will also need to upgrade the `shopify_api` gem to v9.2.0+, in order to remove a deprecated `shopify` command that is contained in that gem.
```

Comment:
Just to tick all the boxes here I think we should give them the upgrade command to run.